Add new format example config file

spxiwh opened the pull request:
This would replace the old-format DQ.ini. This should provide an example of what is needed if running on older data as well.

spxiwh committed:
Add new format example config file

tdent commented on O3exp/pipeline/data_O3_C00.ini:
```diff
@@ -0,0 +1,38 @@
+; http://ligo-cbc.github.io/pycbc/releases/v1.2.0/html/workflow/initialization.html
+h1-channel-name = H1:GDS-CALIB_STRAIN
+l1-channel-name = L1:GDS-CALIB_STRAIN
+v1-channel-name = V1:Hrec_hoft_16384Hz
```
Not clear why V1 is not commented out here as it is in ifos. Add a comment explaining that the V1 options here are ignored unless turned on in ifos.

In general prefer not to have options that are ignored, but convenience may override that here

tdent commented on O3exp/pipeline/data_O3_C00.ini:
```diff
@@ -0,0 +1,38 @@
+datafind-h1-frame-type = H1_HOFT_C00
+datafind-l1-frame-type = L1_HOFT_C00
+datafind-v1-frame-type = V1Online
+;datafind-check-frames-exist = no_test
```
This line is cryptic, remove or explain more

tdent commented on O3exp/pipeline/data_O3_C00.ini:
```diff
@@ -0,0 +1,38 @@
+[workflow-segments]
+segments-vetoes = -CAT_2,-CAT_H
+segments-database-url = https://segments.ligo.org
+
```
Remove empty line...

spxiwh committed:
Update data_O3_C00.ini

spxiwh commented:
Changes made as requested

tdent commented:
Actually I'm still a bit confused as to what this folder is for. In the past we have used it for runs on *O2* data to develop / test for O3. I think that is still its main purpose, though it won't hurt to have an *example* file for O3 which we don't use for production.
So, please make sure there is a viable data.ini for O2 C02 clean data, as this is what our benchmark runs have used up to now.

spxiwh committed:
Adding new O2 ini. Removing old files

spxiwh commented:
@tdent I've added a O2 CO2 data.ini file, and also removed the old files, which now won't work.

I also decided to have Virgo active by default in both the O2 and O3 example ini files.

tdent commented on O3exp/pipeline/data_O3_C00.ini:
```diff
@@ -0,0 +1,35 @@
+datafind-v1-frame-type = V1Online
+
+[workflow-segments]
+segments-vetoes = -CAT_2,-CAT_H
```
as above, minus or plus?

tdent commented on O3exp/pipeline/data_O3_C00.ini:
```diff
@@ -0,0 +1,35 @@
+[workflow-ifos]
+h1 =
+l1 =
+v1=
```
as above, add comment for how to turn off V1

tdent reviewed (changes requested):
thanks, see minor comments and check the veto signs!

spxiwh committed:
Update data_O2_C02.ini

spxiwh committed:
Update data_O3_C00.ini

spxiwh commented:
Argh, I keep getting the `-`s wrong. Should be fixed now.

tdent committed:
space

tdent committed:
space
